src/covers/coverprovider.h

```diff
@@ -35,10 +35,12 @@ class CoverProvider : public QObject {
   Q_OBJECT
 
  public:
-  explicit CoverProvider(const QString& name, QObject* parent);
+  explicit CoverProvider(const QString& name, const bool& fetchall,
```

Comment:
Just use `bool`

Reply:
Doesn't work. All the cover managers won't compile then.